### ISSUE TITLE
How is the code of trick implemented

### ISSUE THREAD

**Issue by Xuxuduoyu** [RaptorMai/online-continual-learning#19]
I want to know the code about trick, but I have not found the code about trick in the file directory.

**Comment by czjghost** [comment]
**please carefully reading this project, all the trick can be traced by following parameters in "[general_main.py](https://github.com/RaptorMai/online-continual-learning/blob/main/general_main.py)".**
    parser.add_argument('--labels_trick', dest='labels_trick', default=False, type=boolean_string,
                        help='Labels trick')
    parser.add_argument('--separated_softmax', dest='separated_softmax', default=False, type=boolean_string,
                        help='separated softmax')
    parser.add_argument('--kd_trick', dest='kd_trick', default=False, type=boolean_string,
                        help='Knowledge distillation with cross entropy trick')
    parser.add_argument('--kd_trick_star', dest='kd_trick_star', default=False, type=boolean_string,
                        help='Improved knowledge distillation trick')
    parser.add_argument('--review_trick', dest='review_trick', default=False, type=boolean_string,
                        help='Review trick')
    parser.add_argument('--ncm_trick', dest='ncm_trick', default=False, type=boolean_string,
                        help='Use nearest class mean classifier')
    parser.add_argument('--mem_iters', dest='mem_iters', default=1, type=int,
                        help='mem_iters')

**BTW, most trick can be found at [base.py](https://github.com/RaptorMai/online-continual-learning/blob/main/agents/base.py) and [kd_manager.py](https://github.com/RaptorMai/online-continual-learning/blob/main/utils/kd_manager.py)**
